### PR TITLE
Create dithereum.json

### PR DESCRIPTION
**Pull request by solidityx**
Create Dithereum Testnet for chain ID 34
added Icons for the Dithereum and Ditherscan

### CODE REVIEW

**Review by ligi — CHANGES_REQUESTED** [review]
Explorer section needs a standard field

**Comment by solidityx** [comment]
> Explorer section needs a standard field

New commits has that changes. And we are waiting for the etherscan like explorer, yet. We can proceed further once it will be up. Thank you.

**Comment by ligi** [comment]
pls ping me when it is up

**Comment by solidityx** [comment]
> pls ping me when it is up
Thanks for your time.. 

Can I remove explorer part for now?  I just want the chain ID to be secured.. and add explorer when its up?  Thanks again.

**Comment by solidityx** [comment]
Removed explorer details for now. hope to get it added now :D
Thanks for your time..

File: _data/icons/dithereum.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+
```
Comment (ligi) [inline _data/icons/dithereum.json:1]:
```suggestion
```

File: _data/icons/dithereum.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+[
+    {
+      "url": "ipfs://QmSHN5GtRGpMMpszSn1hF47ZSLRLqrLxWsQ48YYdJPyjLf", 
+      "width": 500,
+      "height": 500,
+      "format": "png"
+    }
+]
+
```
Comment (ligi) [inline _data/icons/dithereum.json:10]:
```suggestion
```

File: _data/chains/eip155-34.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  "nativeCurrency": {
+    "name": "Dither",
+    "symbol": "DTH",
+    "decimals": 18
+  },
+  "infoURL": "https://dithereum.org",
+  "shortName": "dth",
+  "chainId": 34,
+  "networkId": 34,
+  "ens": {}
```
Comment (ligi) [inline _data/chains/eip155-34.json:21]:
```suggestion
```

File: _data/chains/eip155-34.json [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+  ],
+  "nativeCurrency": {
+    "name": "Dither",
+    "symbol": "DTH",
+    "decimals": 18
+  },
+  "infoURL": "https://dithereum.org",
+  "shortName": "dth",
+  "chainId": 34,
+  "networkId": 34,
```
Comment (ligi) [inline _data/chains/eip155-34.json:20]:
```suggestion
  "networkId": 34
```

**Review by Hawkins37 — APPROVED** [review]

**Pull request merged**